src/api/app/assets/stylesheets/webui/application/comments.scss

```diff
@@ -17,9 +17,6 @@
   border-top: none;
 }
 
-.comment-actions {
```

Comment:
I think, he means "remove it from _links.html" 

Coolo, translating between OBS team members since 2008 ;)